CMake/SiloConfig.cmake.in

```diff
@@ -0,0 +1,79 @@
+#-----------------------------------------------------------------------------
+# SILO Config file for compiling against SILO build directory
+#-----------------------------------------------------------------------------
+@PACKAGE_INIT@
+
+set(SILO_PACKAGE_NAME "Silo")
+
+set (${SILO_PACKAGE_NAME}_VALID_COMPONENTS static shared)
+
+#-----------------------------------------------------------------------------
+# User Options
+#-----------------------------------------------------------------------------
+set (SILO_BUILD_SHARED_LIBS    @BUILD_SHARED_LIBS@)
+
+set(SILO_ENABLE_SILEX @SILO_ENABLE_SILEX@)
+set(SILO_ENABLE_BROWSER @SILO_ENABLE_BROWSER@)
+set(SILO_ENABLE_FORTRAN @SILO_ENABLE_FORTRAN@)
+set(SILO_ENABLE_HDF5 @SILO_ENABLE_HDF5@)
+set(SILO_ENABLE_JSON @SILO_ENABLE_JSON@)
+set(SILO_ENABLE_PYTHON_MODULE @SILO_ENABLE_PYTHON_MODULE@)
+set(SILO_ENABLE_NETCDF @SILO_ENABLE_NETCDF@)
+set(SILO_ENABLE_TESTS @SILO_ENABLE_TESTS@)
+set(SILO_ENABLE_TESTS_INSTALL @SILO_ENABLE_TESTS_INSTALL@)
```

Comment:
What does `TESTS_INSTALL` do? Will it install all the exe's for the tests? I don't recall doing/needing that in the past. And, all the test exes would be large-ish in disk space. But, a small subset could be useful. 

Thoughts?

Reply:
It was my way of capturing that you wanted a mode for running tests against an install (?) Perhaps I misunderstood. Nothing done in the way of implementation, so I can remove this.